region/maridia/inner-yellow.json

```diff
@@ -149,6 +157,15 @@
                 "requires": null,
                 "openEnd": 1
               }
+            ],
+            "canLeaveCharged": [
+              {
+                "usedTiles": 22,
+                "framesRemaining": 0,
+                "shinesparkFrames": 30,
+                "openEnd": 2,
+                "requires": [ "canSwim", "Morph" ]
```

Comment:
Why gravity? If you time it right the water won't matter at the morph gap, and I can't find a way that goes deeper in the water and still has time to get it off, even with Gravity.

Reply:
Because it's such a very tight maneuver even without taking the water into account. I can see large amounts of time being spent here if you need to time for water as well.

Reply:
It's roughly a 4-second cycle. You really only have to know when to start. and then it's the same execution.
But yeah, I'm a bit concerned that many of those strats require tight execution and aren't locked behind any kind of tech.

Reply:
At least it's locked behind managing 22 tile charges.

Reply:
Beats having nothing, but the ability to do a 2-tap isn't closely related to executing that spark.